db/migrate/20140110000001_update_environments_add_katello_id.rb

```diff
@@ -1,10 +0,0 @@
-class UpdateEnvironmentsAddKatelloId < ActiveRecord::Migration[4.2]
-  def up
-    add_column :environments, :katello_id, :string, :limit => 255
-    add_index :environments, :katello_id
-  end
-
-  def down
-    remove_column :environments, :katello_id
-  end
-end
```

Comment:
does it make more sense to leave the migration file as is but change was it does?

Reply:
You tell me :)
I can probably just have a check there `if table_exists?` that way we wouldn't need to worry about keeping the migration version is `schema_migrations` table.

Reply:
I think it does, i'm not sure of the consequences of deleting a migration that Rails thinks was migrated.  And yeah a check of table_exists? i think is how we handled this stuff in the past

Reply:
I went with doing nothing, as we don't want the `katello_id` on environments.
I have https://github.com/theforeman/foreman_puppet/pull/200 to remove it for users who continue using puppet.